tensorboard/uploader/exporter_test.py

```diff
@@ -199,6 +200,61 @@ def stream_experiments(request, **kwargs):
     self.assertIn("../authorized_keys", msg)
     mock_api_client.StreamExperimentData.assert_not_called()
 
+  def test_fails_nicely_on_stream_experiment_data_timeout(self):
+    # Setup: Client where:
+    #   1. stream_experiments will say there is one experiment_id.
+    #   2. stream_experiment_data will raise a grpc CANCELLED, as per
+    #      a timeout.
+    mock_api_client = self._create_mock_api_client()
+    experiment_id="123"
+
+    def stream_experiments(request, **kwargs):
+      del request  # unused
+      yield export_service_pb2.StreamExperimentsResponse(
+          experiment_ids=[experiment_id])
+
+    def stream_experiment_data(request, **kwargs):
+      raise test_util.grpc_error(grpc.StatusCode.CANCELLED, "details string")
```

Comment:
nit: <code>del request &nbsp;# unused</code> for consistency?

Reply:
Done.